src/odemis/acq/test/move_test.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright © 2020 Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import logging
+import os
+import time
+import unittest
+
+import odemis
+from odemis import model
+from odemis.acq.move import LOADING, IMAGING, getLoadingProgress, RTOL_PROGRESS
+from odemis.acq.move import cryoTiltSample, cryoLoadSample
+from odemis import util
+from odemis.util import test
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
+CRYO_SECOM_CONFIG = CONFIG_PATH + "sim/cryosecom-sim.yaml"
+
+ATOL_STAGE = 1e-7
+
+
+class TestCryoMove(unittest.TestCase):
+    """
+    Test cryoLoadSample and cryoTiltSample functions
+    """
+    backend_was_running = False
+
+    @classmethod
+    def setUpClass(cls):
+        try:
+            test.start_backend(CRYO_SECOM_CONFIG)
+        except LookupError:
+            logging.info("A running backend is already found, skipping tests")
+            cls.backend_was_running = True
+            return
+        except IOError as exp:
+            logging.error(str(exp))
+            raise
+
+        # find components by their role
+        cls.stage = model.getComponent(role="stage")
+        cls.focus = model.getComponent(role="focus")
+
+        cls.stage_active = cls.stage.getMetadata()[model.MD_FAV_POS_ACTIVE]
+        cls.focus_deactive = cls.focus.getMetadata()[model.MD_FAV_POS_DEACTIVE]
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.backend_was_running:
+            return
+        test.stop_backend()
+
+    def setUp(self):
+        if self.backend_was_running:
+            self.skipTest("Running backend found")
+
+    def test_loading(self):
+        """
+        Test moving the sample stage to loading position while it's in imaging position
+        """
+        stage = self.stage
+        # Get the stage to imaging position
+        f = stage.moveAbs(self.stage_active)
```

Comment:
On the real hardware, that's kind of dangerous to do.... and it's exactly why we are designing a special function ;-)
I'd recommend to merge `test_loading` and `test_imaging_from_loading`. In it, start with a call to cryoLoadSample(LOADING), do some basic tests, call cryoLoadSample(IMAGING), more checks, and then  cryoLoadSample(LOADING) with full checks.